crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -36,6 +36,11 @@ def restore_background
       def restore
         cleanup if self.class.restore_steps_path.exist?
 
+        # disable/stop dns service before starting restore
+        self.class.disable_dns_path.open("a") do |f|
+          f.write "#{Time.zone.now.iso8601}\n"
```

Comment:
maybe we could even add the name of the backup from which the restore was triggered

and why do you append instead of just "w"?